jobs/keepalived/templates/bin/post-start.erb

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+	
+set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+# source properties file here.
+source /var/vcap/jobs/keepalived/bin/job_properties.sh
+export IS_HA=<%= link("broker").p("ha_enabled") %>
+iaas_name='<%= p('iaas_config.name', nil) %>'
+
+if [[ "$iaas_name" == "gcp" && "$IS_HA" == "true" ]]; then
+  echo "post-start script execution for gcp in-progress"	
+	
+  # All variables used here are defined and exported from job_properties.sh file.
+
+  exec /var/vcap/packages/ha-utils/bin/iaas-utils ${AUTHBASEURL} ${PRIVATEKEYID} ${CLIENTEMAILID} ${GCPBASEURL} ${PROJECTID} ${NETWORK} ${SUBNET} ${DEPLOYMENTGUID} ${FLOATINGIP} ${INSTANCEIP} ${SFBROKERPORT} ${SFREPORTPORT} ${SFEXTERNALPORT} ${SFDEPHOOKSPORT} ${PROBEINTERVAL} ${PROBEHEALTHCHECK} ${PROBEPROTOCOL} ${PROBEPORT} ${PROBEREQUESTPATH} ${LANDSCAPE} "${PRIVATEKEY}" ${SCOPES}
+	
+  RETURN_CODE=$?
+		
+  echo "$AUTHBASEURL"
```

Comment:
Still feel that these echo's will be unnecessary in the production. We can remove it IMO.

Reply:
Removed the echos!